CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing Guide
+
+This repository is a collection of best practices and coding standards. It has been made by [research
+software engineers] at [UCL's Centre for Advanced Research Computing][UCL ARC]. Some of it is from our internal
+documentation and recommendations pulled out and made public. The content is stored in [GitHub flavoured markdown], and
+is rendered as a live [webpage].
+
+Any changes should be reviewed by a member or associate member of ARC. If the additions are taken from ARC's internal
+documentation, **please add the original author** as a reviewer to check that they're OK with their content being made
+public a CC-by-4.0. If something seems like it's broken, please go ahead and [open an issue]!
+
+> [!NOTE] We're open for contributions from anyone but it's the UCL ARC recommended best-practices, so ulitmately it
```

Comment:
```suggestion
> [!NOTE]
> We're open for contributions from anyone but it's the UCL ARC recommended best-practices, so ulitmately it
```

Is this my fault or one of the `pre-commit` hook's?

Reply:
@paddyroddy can haz some help plz.
`prettier` is breaking this.

The [GFM syntax for a callout needs a newline](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) here:

```
> [!NOTE]
> This will be rendered.
```

```
> [!NOTE] This will not
```

Reply:
If you view the file at 2ab2120 it's fine. If you see c338f1d it's broken again.

Reply:
OK, it's reported upstream:

- https://github.com/prettier/prettier/issues/15479